lib/libefi/rdwr_efi.c

```diff
@@ -399,8 +399,10 @@ efi_alloc_and_init(int fd, uint32_t nparts, struct dk_gpt **vtoc)
 	length = sizeof (struct dk_gpt) +
 	    sizeof (struct dk_part) * (nparts - 1);
 
-	if ((*vtoc = calloc(1, length)) == NULL)
+	if ((*vtoc = calloc(1, length)) == NULL) {
+		// cppcheck-suppress memleak
```

Comment:
This suppresses the warning, but let's check this reported leak against the Coverity analysis for the same file.  It should provide a more detailed analysis about the leak if it's in fact real.